src/Front/Parser.y

```diff
@@ -35,10 +35,12 @@ import Common.Compiler (ErrorMsg)
   'let'   { Token (_, TLet) }
   'after' { Token (_, TAfter) }
   'wait'  { Token (_, TWait) }
-  'fun'   { Token (_, TFun) }
```

Comment:
Have you merged from `main` recently? This should not have been deleted. Please do not merge until this is fixed.

Reply:
nvm I just fixed this